src/baldr/geojson.cc

```diff
@@ -19,14 +19,13 @@ template <class coord_t>
 MapPtr to_geojson(const typename midgard::GriddedData<coord_t>::contours_t& grid_contours, bool polygons, const std::vector<std::string>& colors) {
   //for each contour interval
   int i = 0;
-  auto color_itr = colors.cbegin();
+  auto color_itr = colors.crbegin();
```

Comment:
Ya that makes sense. We will have to somehow associate the color with the time value which might be a bit complicated or just take some api fixes and stuff. But currently if the time values are out of order in your query, loki rejects the query completely. So my fix works, it just won't if we decide to release that restriction.